pkg/driver/sanity_test.go

```diff
@@ -152,6 +153,24 @@ func (p *fakeCloudProvider) GetPVMInstanceByID(instanceID string) (*cloud.PVMIns
 	}, nil
 }
 
+func (p *fakeCloudProvider) GetPVMInstanceDetails(instanceID string) (*models.PVMInstance, error) {
+
+	return &models.PVMInstance{
+		PvmInstanceID: &instanceID,
+		ServerName:    &strings.Split(instanceID, "-")[0],
```

Comment:
this indexing 0 seems to be little fishy to me, prefer creating a different issue to remove such indexing from the code cc @Madhan-SWE

Reply:
Have updated with minimal changes as of now, will leave it upto @Madhan-SWE to fix it as part of new issue (specific)